aips/aip-lite-account.md

```diff
@@ -0,0 +1,176 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Introduce Lite Account
+author: lightmark, davidiw
+Status: Draft
+type: Framework
+created: <06/25/2024>
+---
+
+# AIP-X - Introduce Lite Account
+  
+## Summary
+As Aptos evolves, deficiencies in our early account model become more apparent. These limitations prevent efficient usage of resources on Aptos and hamper evolution of new features. For example, Aptos is the only blockchain that
+requires an account to be populated prior to use and assets can only be allocated to an existing account. This AIP reviews many distinct initiatives that fall into the umbrella of redesiging accounts that seek to rewrite
+the entirety of the user experience.
+
+This AIP proposes a new account model to replace the current account model on Aptos in the future, called `LiteAccount` that leverages the composability of Aptos Object model, which solves all the issues and cleans up all unnecessary resources.
+Also, a migration plan is included in the AIP.
+
+### Out of Scope
+
+The concrete plan to remove the only required resource, sequence number.
+
+## High-level Overview
+
+Lite Account will have all account related resources into the same `PrimaryFungibleStore`(PFS) object group at the PFS object address.
+The new struct will be in `account.move`, previously fields in account v1. The optionality of a resource is determined by its existence. To be compatible with account v1, some fields will be converted to the counterpart in lite account if the lite account is migrated from v1.
+The potential resources are:
```

Comment:
Could we add another type called `lock` or `resource_lock` (programmable permissions that are either valid for a specified time or throughout the lifecycle of an intent)?

This would align well with the [Aptos Intents Framework AIP](https://github.com/aptos-foundation/AIPs/pull/511/files). @lightmark

Reply:
@ch4r10t33r take a look at https://github.com/aptos-foundation/AIPs/pull/510